### PR TITLE
Update rubygem-fog-google to 1.11.0

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Includes the dependencies (hopefully :smile:)

Needed by: theforeman/foreman#7242

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
No idea what obal linter wants from me :thinking:

**Comment by ezr-ondrej** [comment]
:green_apple: :tada:

File: packages/foreman/rubygem-google-cloud-env/rubygem-google-cloud-env.spec [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.yardopts
+%{gem_instdir}/CODE_OF_CONDUCT.md
```
Comment (ekohl) [inline packages/foreman/rubygem-google-cloud-env/rubygem-google-cloud-env.spec:79]: We typically exclude this.

Comment (ezr-ondrej) [reply packages/foreman/rubygem-google-cloud-env/rubygem-google-cloud-env.spec]: :+1: done

**Review by zjhuntin — APPROVED** [review]
ACK

**Pull request merged**